src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -240,5 +240,53 @@ <H2>Address Cache</H2>
 <P>Since these 2 properties are part of the security policy, they are
 not set by either the -D option or the {@code System.setProperty()} API,
 instead they are set as security properties.</P>
+<a id="Unixdomain"></a>
+<H2>Unix domain sockets</H2>
+<P>There are a number of system (and networking) properties that affect the behavior of
+channels to <i>Unix domain</i> server sockets when binding <i>automatically</i>.
```

Comment:
Should this also mention <i>implicit</i> binding?

Reply:
I'm not sure, as the system properties don't affect implicit binding, but I would have liked to have a place to "explain" all aspects of binding of Unix domain sockets/server-sockets. Maybe, I could add a paragraph at the end, just as an aside or note.

Reply:
These concepts aren't new by the way. "Implicit" binding is when you create a client socket and connect it without calling bind first. This is not possible with server sockets.

"Automatic" binding is when you bind a server socket with a null address. All the above is the same for inet and unix domain. inet has more options relating to the interface and specifying a port = 0. These cases don't exist for unix domain